src/PhpParser/NodeVisitor/StringScalarPrefixer.php

```diff
@@ -107,6 +108,8 @@ private function prefixStringScalar(String_ $string): String_
             return $string;
         }
 
+        $normalizedValue = '\\' === $string->value[0] ? substr($string->value, 1) : $string->value;
```

Comment:
Wouldn't an `ltrim` be cleaner here?

Reply:
e.g. `$normalizedValue  = ltrim($string->value, '\\');`

Reply:
Indeed, much better